libs/pika/async_mpi/include/pika/async_mpi/transform_mpi.hpp

```diff
@@ -85,6 +85,8 @@ namespace pika::mpi::experimental {
                         return ex::schedule(default_pool_scheduler(p));
                     }
                     return just(request) | trigger_mpi(mode) | transfer(default_pool_scheduler(p));
+                    return just(request) | trigger_mpi(mode) |
+                        ex::continues_on(default_pool_scheduler(p));
```

Comment:
This looks suspicious.

Reply:
in what way?

Reply:
Sorry, should've marked more context. Look at line 87. The return statement is duplicated.

Reply:
oops. Edited it in conflict editor, obviously saved it badly. fixed